### PR TITLE
feature/TOOL-7507-GPSLocator

### PR DESCRIPTION
**Pull request by Richardj52**
Now not only will you able to get an estimate based on typing the address on the pickup and destination text boxes, you can also now click on the map where you would like to get picked up and dropped off at and will be able to get an estimate.

### CODE REVIEW

**Comment by dmssargent** [comment]
@jusoto The bulk of the changes seems to be with package-lock.json, should this file be rm'ed and gitignored?

File: src/app/common/components/presentation/text-box/text-box.component.ts [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+
   @Input() private id: string;
   @Input() private placeholder: string;
   @Input() private encryption: string;
   @Input() private icon: string;
   @Input() private textboxValue: string;
+  ngOnChanges(changes: SimpleChanges): void {}
 
-  @Output() textboxChange: EventEmitter<String> = new EventEmitter<String>();
+  //@Output() textboxChange: EventEmitter<String> = new EventEmitter<String>();
```
Comment (dmssargent) [inline src/app/common/components/presentation/text-box/text-box.component.ts]: Do we need still need this even though it is commented out?

File: src/app/passengers/components/smart/trip-planner/trip-planner.component.html [hunk 2]
```diff
@@ -1,25 +1,23 @@
 <mat-card>
   <h1>{{ title }}</h1>
   <div class="container">
-    <agm-map [latitude]="latitude" [longitude]="longitude" [zoom]="zoom">
+    <agm-map [latitude]="latitude" [longitude]="longitude" [zoom]="zoom" (mapClick)="getGeoCode($event)">
       <div appDirectionsService></div>
     </agm-map>
   </div>
-  <div class="inprogress">
+  <!-- <div class="inprogress">
```
Comment (dmssargent) [inline src/app/passengers/components/smart/trip-planner/trip-planner.component.html]: Do we need this commented out code?

File: src/app/common/components/presentation/text-box/text-box.component.ts (hunk 1 above)
Comment (Richardj52) [reply src/app/common/components/presentation/text-box/text-box.component.ts]: this has been addressed & removed on my recent push just made now

File: src/app/passengers/components/smart/trip-planner/trip-planner.component.html (hunk 2 above)
Comment (Richardj52) [reply src/app/passengers/components/smart/trip-planner/trip-planner.component.html]: this has been addressed & removed on my recent push just made now

**Review by dmssargent — APPROVED** [review]

**Review by jschwait — APPROVED** [review]

**Pull request merged**